packages/wonder-blocks-toolbar/src/components/toolbar.tsx

```diff
@@ -73,74 +67,70 @@ type DefaultProps = {
  * />
  * ```
  */
-export default class Toolbar extends React.Component<Props> {
-    static defaultProps: DefaultProps = {
-        color: "light",
-        leftContent: null,
-        rightContent: null,
-        size: "medium",
-    };
-
-    render(): React.ReactNode {
-        const {color, leftContent, rightContent, size, subtitle, title} =
-            this.props;
-
-        const TitleComponent = subtitle ? LabelLarge : HeadingSmall;
+export default function Toolbar({
```

Comment:
**note:** Took the opportunity to convert this to a functional component. Most of the code remains the same (except for the custom title part).